networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/reputation/ReputationManager.java

```diff
@@ -13,156 +13,32 @@
 
 package tech.pegasys.teku.networking.p2p.reputation;
 
-import com.google.common.base.MoreObjects;
-import java.util.EnumSet;
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import org.hyperledger.besu.plugin.services.MetricsSystem;
-import tech.pegasys.teku.infrastructure.collections.cache.Cache;
-import tech.pegasys.teku.infrastructure.collections.cache.LRUCache;
-import tech.pegasys.teku.infrastructure.metrics.TekuMetricCategory;
-import tech.pegasys.teku.infrastructure.time.TimeProvider;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 import tech.pegasys.teku.networking.p2p.network.PeerAddress;
 import tech.pegasys.teku.networking.p2p.peer.DisconnectReason;
-import tech.pegasys.teku.networking.p2p.peer.NodeId;
 
-public class ReputationManager {
+public interface ReputationManager {
   // This is not a big ban, we expect the peer could be usable very soon
```

Comment:
We can add a `NOOP` value here similar to BlobSidecarManager. Even though it won't be used anywhere, It could be used easily to switch implementations.